airflow/providers/google/cloud/hooks/datafusion.py

```diff
@@ -465,7 +474,98 @@ def stop_pipeline(self, pipeline_name: str, instance_url: str, namespace: str =
             "DataPipelineWorkflow",
             "stop",
         )
+        self.log.info("before stp pipe")
```

Comment:
Is that the intended logging message? Or leftover from debugging.

Reply:
Debugging :D